core/keyboard_nav/navigation.js

```diff
@@ -653,127 +654,10 @@ Blockly.Navigation.handleEnterForWS = function() {
  * @return {!boolean} True if the key was handled false otherwise.
  */
 Blockly.Navigation.navigate = function(e) {
```

Comment:
Should probably rename this to onKeypress or something similar since it's handling keypress events.

Reply:
done